EdgeOfEmpireBot/EdgeOfEmpireBot/BotApp.cs

```diff
@@ -139,20 +151,29 @@ private static async Task MessageUpdated(Cacheable<IMessage, ulong> before, Sock
     /// <returns>the token string</returns>
     private static string GetToken()
     {
-        var token = string.Empty;
-
-        try
-        {
-            string path = Path.Combine("Data/token.txt");
-            token = File.ReadAllText(path);
-        }
-        catch (Exception ex)
+        // Attempt to get the token from an environment variable named "DISCORD_TOKEN"
+        var token = Environment.GetEnvironmentVariable("DISCORD_TOKEN");
+        Console.WriteLine(token);
+        // If the environment variable is not set or is empty, attempt to read from the file
+        if (string.IsNullOrEmpty(token))
         {
-            Console.WriteLine(ex.ToString());
+            token = string.Empty;
+            try
+            {
+                string path = Path.Combine("Data/token.txt");
+                token = File.ReadAllText(path);  // Fallback to reading the token from the file if the environment variable is not set
+            }
+            catch (Exception ex)
+            {
+                token = string.Empty;
```

Comment:
why are we making the token empty here if it already failed to be Read successfully?